web-portal/components/DatePicker.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div>
+    <flat-pickr readonly="true" v-model="date" :config="config"></flat-pickr>
+  </div>
```

Comment:
Is the wrapping `div` necessary?

Reply:
I was wondering that myself. I hadn't gotten around to testing what happens without it

Reply:
removed

Reply:
https://github.com/infinite-industries/infinite/commit/26c88da4454a1de00277cade533e92664483984d